### PR TITLE
Add acrechain info

### PR DESCRIPTION
**Pull request by gophereth**
- Add acrechain info
- Add IBC transfer channel open for Osmosis <> Acrechain
- Add acrechain logo

### CODE REVIEW

File: packages/web/config/chain-infos.ts [hunk 1]
```diff
@@ -1872,6 +1872,34 @@ const chainInfos = (
...
+          isFeeCurrency: true,
+        },
+      ],
+      gasPriceStep: {
+        low: 10000000000,
+        average: 25000000000,
+        high: 40000000000,
+      },
+      features: ["ibc-transfer", "ibc-go", "eth-address-gen", "eth-key-sign"],
+      explorerUrlToTx: "https://cosmosrun.info/acre-mainnet/tx/${txHash}",
```
Comment (JeremyParish69) [inline packages/web/config/chain-infos.ts:1901]: please remove the '$' dollar sign.

File: packages/web/config/ibc-assets.ts [hunk 2]
```diff
@@ -784,6 +784,13 @@ export const IBCAssetInfos: (IBCAsset & {
     //   "https://blue.kujira.app/ibc?destination=kaiyo-1&source=osmosis-1&denom=ukuji",
     isVerified: true,
   },
+  {
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:787]: please move to the bottom of the list

File: packages/web/config/ibc-assets.ts [hunk 3]
```diff
@@ -784,6 +784,13 @@ export const IBCAssetInfos: (IBCAsset & {
...
+    counterpartyChainId: "acre_9052-1",
+    sourceChannelId: "channel-490",
+    destChannelId: "channel-0",
+    coinMinimalDenom: "aacre",
+    isVerified: true,
```
Comment (JeremyParish69) [inline packages/web/config/ibc-assets.ts:792]: please remove the 'isVerified' property (this is used to display on Osmosis Main, but this asset has not yet been approved.

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
Please see inline comments

File: packages/web/config/chain-infos.ts (hunk 1 above)
Comment (gophereth) [reply packages/web/config/chain-infos.ts]: Done - https://github.com/osmosis-labs/osmosis-frontend/pull/1085/commits/3fe56229141767847e9cf12660ae72714dea7d1f

File: packages/web/config/ibc-assets.ts (hunk 2 above)
Comment (gophereth) [reply packages/web/config/ibc-assets.ts]: Done https://github.com/osmosis-labs/osmosis-frontend/pull/1085/commits/3fe56229141767847e9cf12660ae72714dea7d1f

File: packages/web/config/ibc-assets.ts (hunk 3 above)
Comment (gophereth) [reply packages/web/config/ibc-assets.ts]: Done - https://github.com/osmosis-labs/osmosis-frontend/pull/1085/commits/3fe56229141767847e9cf12660ae72714dea7d1f

**Review by JeremyParish69 — APPROVED** [review]
LGTM

**Pull request merged**